test/components/select-test.jsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+var React = require('react');
+var TestUtils = require('react-addons-test-utils');
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var SelectView = require('../../src/components/select/select.jsx');
+
+describe('SelectComponent', function() {
+  it('should render select with label', function() {
+    var select = TestUtils.renderIntoDocument(
+      <SelectView label="test" />
+    );
+
+    var renderedSelect = TestUtils.findRenderedDOMComponentWithClass(select, 'component-select-label');
+    assert.isDefined(renderedSelect);
+
+  });
+
+  it('should not have default value', function() {
+    var select = TestUtils.renderIntoDocument(
```

Comment:
I could but testing that it has no attributes is not something I'm currently testing